### DISCUSSION TITLE
Looking to contribute

### DISCUSSION THREAD

**Discussion by dessygil** [datamol-io/molfeat#36] [Ideas]
Hi all, 

I am looking to contribute to Molfeat and wanted to know if there were any plugins you were specifically looking to expand into that I could try. I'm open to all suggestions!

Thanks for any ideas in advance.

**Comment by maclandrol** [comment]
Hello @dessygil,

Thanks for your interest,  we welcome any plugin or any contribution to the main package. 

If you are looking for any idea, please see this list: https://github.com/junxia97/awesome-pretrain-on-molecules maintained by @junxia97. 

You will likely require the pretrained weights for the model you picked, but also assess the original authors codebase and licensing information. Happy to help.

**Reply by dessygil** [reply]
Awesome, thanks for the response. Looking forward to carrying on our conversation over Slack :)
